embassy-lora/Cargo.toml

```diff
@@ -9,13 +9,15 @@ src_base = "https://github.com/embassy-rs/embassy/blob/embassy-lora-v$VERSION/em
 src_base_git = "https://github.com/embassy-rs/embassy/blob/$COMMIT/embassy-lora/src/"
 features = ["time", "defmt"]
 flavors = [
+    { name = "rak4631", target = "thumbv7em-none-eabihf",   features = ["rak4631"] },
```

Comment:
Could you rename this feature to sx126x ?

Reply:
Implementation updated to use feature sx126x  and remove rak4631 feature usage in the board specific processing.  I ran into puzzling issues attempting to implement the RadioSwitch functionality, so updated the board specific processing to include antenna handling for the time being.  This will get shaken out when we implement sx12xx processing.

A correction was made to RSSI computation, but that may need more work.  Implementation marked with ??? remains to be investigated.

The two P2P examples have been simplified to support a wider range of nrf/sx126x combinations.

I will be available through October to address discovered issues, but will be unavailable for much of November.  I look forward to working toward an sx12xx solution starting in December.  Thank you for reviewing this submission.